### DISCUSSION TITLE
Text color

### DISCUSSION THREAD

**Discussion by manfredlotz** [zim-desktop-wiki/zim-desktop-wiki#1772] [Q&A] [answered]
I like to mark some text and choose a certain color.

Is this possible? If not are there plans to provide such a feature?

**Comment by Nick222** [comment]
That would be just great ... :)

**Comment by introt** [comment, marked as answer]
No, this is currently not possible. Please see #268 for earlier discussion.

**Reply by manfredlotz** [reply]
Surfing about how colors could be added in markdown I found something like this

```
<span style="color:red">**bold text in blue**</span>
```

Looks like a possibility.
